LibreNMS/Data/Source/Fping.php

```diff
@@ -67,28 +68,10 @@ public function ping($host, $count = 3, $interval = 1000, $timeout = 500, $addre
         $process = app()->make(Process::class, ['command' => $cmd]);
         Log::debug('[FPING] ' . $process->getCommandLine() . PHP_EOL);
         $process->run();
-        $output = $process->getErrorOutput();
-
-        preg_match('#= (\d+)/(\d+)/(\d+)%(, min/avg/max = ([\d.]+)/([\d.]+)/([\d.]+))?$#', $output, $parsed);
-        [, $xmt, $rcv, $loss, , $min, $avg, $max] = array_pad($parsed, 8, 0);
 
-        if ($loss < 0) {
-            $xmt = 1;
-            $rcv = 1;
-            $loss = 100;
-        }
+        $response = FpingResponse::parseOutput($process->getErrorOutput(), $process->getExitCode());
```

Comment:
I see the old code does it also, but fping prints this on stderr?

Reply:
Correct, fun stuff eh?